### PR TITLE
externed sunkit-instruments

### PR DESCRIPTION
**Pull request by nabobalis**
In order to deprecate instr and still have access to it for 3.0. I had to extern sunkit-instruments.

### CODE REVIEW

**Comment by nabobalis** [comment]
Time for an isort AF section?

**Comment by nabobalis** [comment]
Doc build failure due to indentation issues in function docstrings. PRobably need to be patched up stream (even if they pass there).

**Comment by nabobalis** [comment]
Conda failure due to lack of package. That should be fixed before merge.

**Comment by nabobalis** [comment]
I am unclear why RTD is downloading every version of sunpy.

**Comment by nabobalis** [comment]
I might have a circular dependency going on there.

File: sunpy/net/dataretriever/sources/tests/test_rhessi.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 from urllib.request import urlretrieve
 
 import pytest
+import sunkit_instruments.rhessi
```
Comment (Cadair) [inline sunpy/net/dataretriever/sources/tests/test_rhessi.py:7]: we shouldn't have this dep.

Comment (nabobalis) [inline sunpy/net/dataretriever/sources/tests/test_rhessi.py:7]: Remove this.

File: sunpy/net/dataretriever/sources/rhessi.py [hunk 2]
```diff
@@ -7,9 +7,9 @@
 from urllib.request import urlopen, urlretrieve
 
 from dateutil.rrule import MONTHLY, rrule
+from sunkit_instruments import rhessi
```
Comment (Cadair) [inline sunpy/net/dataretriever/sources/rhessi.py:10]: nope :p

**Comment by dstansby** [comment]
Are we doing this for 2.1?

**Comment by nabobalis** [comment]
We pushed this back to 3.0

**Comment by nabobalis** [comment]
Bug in released version of sunkit-instr breaks the CI.

**Comment by nabobalis** [comment]
It has been externed now.

File: docs/conf.py [hunk 3]
```diff
@@ -171,6 +171,7 @@
     "pandas": ("https://pandas.pydata.org/pandas-docs/stable/", None),
     "skimage": ("https://scikit-image.org/docs/stable/", None),
     "drms": ("https://docs.sunpy.org/projects/drms/en/stable/", None),
+    "sunkit_instruments": ("https://docs.sunpy.org/projects/sunkit_instruments/en/stable/", None),
```
Comment (nabobalis) [inline docs/conf.py:174]:
```suggestion
    "sunkit_instruments": ("https://docs.sunpy.org/projects/sunkit-instruments/en/stable/", None),
```

File: sunpy/instr/__init__.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import warnings
+
+from sunpy.util.exceptions import SunpyDeprecationWarning
+
+warnings.warn("sunpy.instr is deprecated and will be removed in sunpy 3.1. Please use sunkit-instruments",
```
Comment (hayesla) [inline sunpy/instr/__init__.py:5]:
```suggestion
warnings.warn("sunpy.instr is deprecated and will be removed in sunpy 3.1. Please use the affiliated sunkit-instruments package",
```

Just to be explicit?

Comment (Cadair) [inline sunpy/instr/__init__.py:5]:
```suggestion
warnings.warn("sunpy.instr is deprecated and will be removed in sunpy 3.1. Please use sunkit-instruments",
```

**Review by hayesla — APPROVED** [review]

File: sunpy/net/dataretriever/sources/tests/test_rhessi.py [hunk 5]
```diff
@@ -80,11 +76,9 @@ def test_get_base_url_on_timeout(mock_urlopen):
         rhessi.get_base_url()
 
 
-# Test `rhessi.get_observing_summary_filename(...)`
-
 def parsed_dbase():
     """
-    The result of calling `parse_observing_summary_dbase_file(...)` on
+    The result of calling `parse_observing_summary_dbase_file...)` on
```
Comment (wtbarnes) [inline sunpy/net/dataretriever/sources/tests/test_rhessi.py:81]: Why remove a `(`?

**Review by wtbarnes — APPROVED** [review]

File: sunpy/net/dataretriever/sources/tests/test_rhessi.py (hunk 5 above)
Comment (nabobalis) [reply sunpy/net/dataretriever/sources/tests/test_rhessi.py:81]: Will fix.

File: sunpy/instr/__init__.py (hunk 4 above)
Comment (nabobalis) [reply sunpy/instr/__init__.py:5]: Will fix.

**Pull request merged**